packages/components/src/KaizenProvider/_docs/KaizenProvider.mdx

```diff
@@ -29,18 +32,20 @@ import { KaizenProvider } from '@kaizen/components';
 
 `KaizenProvider` at this version is a direct replacement for `ThemeProvider`, and is safe to do a 1 to 1 replacement.
 
-<Highlight className="diff">{migrationToV1}</Highlight>
+
+<Highlight className="diff shadow-sm">{migrationToV1}</Highlight>
+
```

Comment:
You are a strange man. You remove blank lines in some docs, yet add double in places like this? 😂